README.md

```diff
@@ -126,6 +127,7 @@ $ docker build . -t <IMAGE_TAG> --build-arg COMMIT=<VERSION_SUFFIX>
 - `--bucket-size`: Specifies the number of the peers can be stored in each bucket.
 - `--tx-quota-per-signer`: Specifies the number of maximum transactions can be included in stage per signer.
 - `--poll-interval`: Specifies the interval between block polling.
+- `--transport-type`: Specifies the type of transport to use.  Either `netmq` or `tcp` is available.  `tcp` by default.
```

Comment:
Could you add `--transport-type` in the usage section (https://github.com/planetarium/NineChronicles.Headless/pull/878/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R20)? 🙏

Reply:
Oops. My bad. Fixed.